<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bots</class>
 <widget class="QDialog" name="Bots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bots</string>
  </property>
  <widget class="QPushButton" name="btnStartStop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtIP">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPort">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbScriptSelection">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Raid Finder</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stars Finder</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Watt Farmer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dump Latest Wild Area Event</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Check Den</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnUpdateProfile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add/Update Profile</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
